### PR TITLE
Advance our scrips and CSVs to 4.9

### PR DESCRIPTION
**Pull request by cynepco3hahue**
Signed-off-by: Artyom Lukianov <alukiano@redhat.com>

### CODE REVIEW

**Comment by yanirq** [comment]
@cynepco3hahue  did you run the bump version script here ?

**Comment by cynepco3hahue** [comment]
> @cynepco3hahue did you run the bump version script here ?

Nope, it only for z-stream.

File: deploy/olm-catalog/performance-addon-operator/4.8.0/performance-addon-operator.v4.8.0.clusterserviceversion.yaml [hunk 1]
```diff
@@ -118,7 +118,7 @@ metadata:
         }
       ]
     capabilities: Basic Install
-    containerImage: REPLACE_IMAGE
+    containerImage: quay.io/openshift-kni/performance-addon-operator:4.8-snapshot
```
Comment (fromanirh) [inline deploy/olm-catalog/performance-addon-operator/4.8.0/performance-addon-operator.v4.8.0.clusterserviceversion.yaml:121]: I'm not sure this is correct
cc @yanirq

File: deploy/olm-catalog/performance-addon-operator/4.8.0/performance-addon-operator.v4.8.0.clusterserviceversion.yaml [hunk 2]
```diff
@@ -237,7 +237,7 @@ spec:
                       fieldPath: metadata.name
                 - name: OPERATOR_NAME
                   value: performance-operator
-                image: REPLACE_IMAGE
+                image: quay.io/openshift-kni/performance-addon-operator:4.8-snapshot
```
Comment (fromanirh) [inline deploy/olm-catalog/performance-addon-operator/4.8.0/performance-addon-operator.v4.8.0.clusterserviceversion.yaml:240]: same

**Review by fromanirh — APPROVED** [review]
/lgtm
/hold

**Pull request merged**